simapp/simd/cmd/testnet.go

```diff
@@ -331,7 +332,7 @@ func collectGenFiles(
 			return err
 		}
 
-		nodeAppState, err := genutil.GenAppStateFromConfig(clientCtx.JSONMarshaler, clientCtx.TxConfig, nodeConfig, initCfg, *genDoc, genBalIterator)
+		nodeAppState, err := genutil.GenAppStateFromConfig(clientCtx.JSONMarshaler, clientCtx.TxConfig, nodeConfig, initCfg, *genDoc, genBalIterator, staking.ValidateMsgInGenesis)
```

Comment:
```suggestion
		nodeAppState, err := genutil.GenAppStateFromConfig(clientCtx.JSONMarshaler, clientCtx.TxConfig, nodeConfig, initCfg, *genDoc, genBalIterator, staking.ValidateGenesisMsg)
```

Reply:
Updated from `ValidateMsgInGenesis` to `ValidateGenesisMsg` :dancers: